oauth2_http/java/com/google/auth/oauth2/PluggableAuthHandler.java

```diff
@@ -112,6 +112,18 @@ public String retrieveTokenFromExecutable(ExecutableOptions options) throws IOEx
       executableResponse = getExecutableResponse(options);
     }
 
+    // If an output file is specified, successful responses must contain the `expiration_time`
+    // field.
+    if (options.getOutputFilePath() != null
```

Comment:
nit: This is a lot of conditions in this statement. IMO could be broken up into variables or a helper function

Reply:
ack